libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/BotFrameworkHttpAdapter.cs

```diff
@@ -139,7 +139,9 @@ private async Task ConnectWebSocketAsync(IBot bot, HttpRequest httpRequest, Http
             try
             {
                 var socket = await httpRequest.HttpContext.WebSockets.AcceptWebSocketAsync().ConfigureAwait(false);
-                var requestHandler = new StreamingRequestHandler(bot, this, socket, _logger);
+                var appId = (_credentialProvider as ConfigurationCredentialProvider)?.AppId;
```

Comment:
Is this assumption correct? what if people register a CertificateCredentialProvider?

Reply:
Changed to access the base adapter.